This is a patch for Gentoo bug 560572, provided by Georgi Georgiev.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 773cf0c..e282ce4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -336,7 +336,7 @@ endif()
 set(VERSION "${PROJECT_VERSION}")
 set(prefix "${CMAKE_INSTALL_PREFIX}")
 set(exec_prefix "\${prefix}")
-set(libdir "\${exec_prefix}/lib")
+set(libdir "${LIB_INSTALL_DIR}")
 set(includedir "\${prefix}/include")
 set(PTHREAD_LIBS "${CMAKE_THREAD_LIBS_INIT}")
 
diff --git a/libical.pc.in b/libical.pc.in
index 7774dce..bf88476 100644
--- a/libical.pc.in
+++ b/libical.pc.in
@@ -8,8 +8,8 @@ iculib=@ICU_LIBRARIES@ @ICU_I18N_LIBRARIES@
 Name: libical
 Description: An implementation of basic iCAL protocols
 Version: @LIBICAL_LIB_VERSION_STRING@
-Libs: -L${libdir} -lical -licalss -licalvcal
+Libs: -lical -licalss -licalvcal
 Libs.private: @PTHREAD_LIBS@
 @REQUIRES_PRIVATE_ICU@
 Cflags: -I${includedir}
